man/udev.xml

```diff
@@ -68,9 +68,10 @@
       pointing to the device node, or run a specified program as part of
       the event handling.</para>
 
-      <para>A rule consists of a comma-separated list of one or more key-value pairs.
-      Each key has a distinct operation, depending on the used operator. Valid
-      operators are:</para>
+      <para>A rule consists of a comma-separated list of one or more key-operator-value pairs.
```

Comment:
pairs → sequences (or something else. But a pair is two items.)

Reply:
Maybe key-operator-value expressions? Trying to avoid "a comma-separated list of ... sequences"